pay-api/src/pay_api/services/auth.py

```diff
@@ -60,21 +60,17 @@ def check_auth(business_identifier: str, account_id: str = None, corp_type_code:
                     'id': f'PASSCODE_ACCOUNT_{business_identifier}'
                 }
         elif account_id:
+            auth_url = current_app.config.get('AUTH_API_ENDPOINT') + f'orgs/{account_id}' \
+                                                                     f'/authorizations?expanded=true'
+            additional_headers = None
             if corp_type_code:
-                auth_url = current_app.config.get('AUTH_API_ENDPOINT') + f'accounts/{account_id}/' \
-                    f'products/{corp_type_code}/authorizations?expanded=true'
-                auth_response = RestService.get(auth_url, bearer_token, AuthHeaderType.BEARER, ContentType.JSON).json()
-                roles: list = auth_response.get('roles', [])
-                if roles:
-                    is_authorized = True
-            else:  # For activities not specific to a product
-                auth_url = current_app.config.get('AUTH_API_ENDPOINT') + f'orgs/{account_id}' \
-                                                                         f'/authorizations?expanded=true'
-
-                auth_response = RestService.get(auth_url, bearer_token, AuthHeaderType.BEARER, ContentType.JSON).json()
-                # TODO Create a similar response as in auth response
+                additional_headers = {'corp-type': corp_type_code}
```

Comment:
may be make the header as product-type or business-type please. Corp Type is actually something we need to refactor on pay side.

Reply:
ok..wil go for product-type